sopel/trigger.py

```diff
@@ -331,6 +343,15 @@ class Trigger(str):
         The :attr:`COMMANDS_WITH_CONTEXT` attribute lists IRC commands for
         which ``sender`` can be relied upon.
 
+    """
+    status_prefix = property(lambda self: self._pretrigger.status_prefix)
+    """The prefix used for the :attr:`sender` for status-specific messages.
+
+    :type: str | None
```

Comment:
I'm too lazy to test it (and it's 01:30 here anyway), but would this work?

```suggestion
    :type: Optional[str]
```

Reply:
Both work.